src/storage/Session.js

```diff
@@ -0,0 +1,38 @@
+/** Pomodoro session class.
+ * @class
+ * @param {string} name - Name of the session.
+ * @property {string} start_time - Starting time of the session
+ * @property {string} end_time - End time of the session
+ * @property {number} nb_work_periods - Number of work periods
+ * @property {number} break_duration - Break duration in minutes
+ * @property {number} work_period_duration - Work period duration in minutes
+ */
+class Session {
+  constructor(name) {
+    // ...
+  }
+
+  /**
+   * End a session.
+   * @return {boolean} True if success else False
+   */
+  end() {
+  // ...
+  }
+
+  /**
+   * Store the session in persistent data.
+   * @return {number} id of the session if success.
+   */
+  store() {
+  // ...
+  }
+
+  /**
+   * Get the session in persistent data.
+   * @return {number} id of the session if success.
+   */
+  getFromLocalStorage() {
+    // ..
+  }
```

Comment:
Unclear what this does - not sure if it like exports the session state or what. Could you elaborate in the docs a bit?

Reply:
I just removed this function as I am not sure of its usefulness.